bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerV2Handler.java

```diff
@@ -145,7 +145,7 @@ protected State getValue(String channelId, GoEStatusResponseBaseDTO goeResponseB
             case ALLOW_CHARGING:
                 return goeResponse.allowCharging == true ? OnOffType.ON : OnOffType.OFF;
             case TEMPERATURE_TYPE2_PORT:
-                if (goeResponse.temperatures == null) {
+                if (goeResponse.temperatures == null || goeResponse.temperatures.length < 2) {
```

Comment:
Why not 1 instead of 2 ?
The idea is to be sure to have at least one value to then get it with `goeResponse.temperatures[0]` ?

Reply:
Hello,

in my case it was like this. If the Array Length if goeResponse.temperatures was 2 there were realistic values for the temperatures. If it was only one  value in the array the value was far away from possible values (e.g. -16843.01). That was the reason why I would ignore such values.

Reply:
Ok, could you please add a comment in the code to explain that ?